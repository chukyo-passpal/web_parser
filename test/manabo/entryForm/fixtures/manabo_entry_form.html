<script>
    $(function () {
        // event setup
        $('.button-send-entry').off().on('click', function (e) {
            glexa.ajaxForm({
                form: '#form-entry',
                method: 'get',
                onSuccess: function () {
                    glexa.closeRemoteModal();
                    glexa.alert('出席を受け付けました');
                    isClassEntryOpened = false;
                }
            });
        });

        // first load
    });
</script>

<div class="modal-dialog modal-lg">
    <div class="modal-content">
        <div class="modal-header">
            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><i
                    class="glyphicon glyphicon-remove"></i></button>
            <h4 class="modal-title">出欠カード</h4>
        </div>
        <div class="modal-body">
            <div class="form-inline">
                <form id="form-entry">
                    <input type="hidden" name="action" value="glexa_modal_entry_form_accept" />
                    <input type="hidden" name="class_id" value="85141" />
                    <input type="hidden" name="directory_id" value="0" />
                    <input type="hidden" name="entry_id" value="45263" />
                    <input type="hidden" name="uniqid"
                        value="9b1e9f349a4cbac489d766f93e8cc96b21e4b559d45097b97bc4c82370993158" />

                    <p>
                        出席を受付中です
                    </p>
                    <p>出席ボタンをクリックしてください</p>
                </form>
            </div>
        </div><!--modal-body-->
        <div class="modal-footer">
            <button class="btn btn-primary button-send-entry size10">出席</button>
        </div>
    </div><!--modal-content-->
</div><!--modal-dialog-->
