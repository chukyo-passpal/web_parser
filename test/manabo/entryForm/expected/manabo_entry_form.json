{
  "action": "glexa_modal_entry_form_accept",
  "classId": "85141",
  "directoryId": "0",
  "entryId": "45263",
  "uniqid": "9b1e9f349a4cbac489d766f93e8cc96b21e4b559d45097b97bc4c82370993158",
  "messages": [
    "出席を受付中です",
    "出席ボタンをクリックしてください"
  ]
}